zepto-dnd.js

```diff
@@ -661,6 +730,9 @@
   }
 
   Sortable.prototype.over = function(e) {
+    if (this.scroller)
+      this.scroller.scrollEvent(this.el, e);
+
```

Comment:
I am doing some adjustments to your code (e.g. pulling some scroll stuff from touch-dnd over). So amending the PR is not required, a quick test would be fine, thanks!